Screenshot To Text/README.md

```diff
@@ -0,0 +1,30 @@
+# OCR Text Extractor
```

Comment:
Please add this script to the main README file.  
`Screenshot To Text/main.py`. Why is this empty?  
`Selfie with Python/README.md`, why is this empty as well?

Reply:
I resolved that now. could you please check